.github/workflows/ci.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
         shell: "bash -l {0}"
         run: |
           conda activate env
-          python -m pip install "zfpy>=1" "numpy<2"
```

Comment:
This is needed to avoid `zfpy` trying to install a very old version of numpy. Instead, by installing the extra, it respects the minimum version of numpy supported by numcodecs.